eng/common/Update-Change-Log.ps1

```diff
@@ -1,153 +0,0 @@
-# Note: This script will add or replace version title in change log
```

Comment:
Lets not delete this script under eng/common as part of this PR. Lets do that from the tools repo as a follow-up.

Reply:
Oh yes. That's right.